docs/content/en/docs/fleet-manager/distributedstorage-plugin.md

```diff
@@ -0,0 +1,416 @@
+---
+title: "Enable Policy Management with fleet"
+linkTitle: "Policy Management"
+weight: 30
+description: >
+  The easiest way to enable policy management with fleet.
+---
+
+In this tutorial, we will cover how to implement unified distributed storage on a set of clusters, using [Fleet](https://kurator.dev/docs/references/fleet-api/#fleet)
+
+## Architecture
+
+Fleet's unified distributed storage is built on top of [Rook](https://rook.io/), and the overall architecture is shown below:
+
+{{< image width="100%"
+    link="./image/fleet-policy.drawio.svg"
+    >}}
+
+## Prerequisites
+
+### 1. Fleet Manager Setup
+
+Set up the Fleet manager by following the instructions in the [installation guide](/docs/setup/install-fleet-manager/).
+
+### 2. Rook prerequisites
+
+1. Kubernetes v1.22 or higher is supported
+1. To configure the Ceph storage cluster, at least one of these local storage types is required:
+
+   - Raw devices (no partitions or formatted filesystems)
+   - Raw partitions (no formatted filesystem)
+   - LVM Logical Volumes (no formatted filesystem)
+   - Persistent Volumes available from a storage class in `block` mode
+
+The easiest way to do this is to mount an Raw disk on the nodes.
+
+### 3. Secrets and Setup for Attached Clusters
+
+1. Running the following command to create two secrets to access attached clusters.
+
+```console
+kubectl create secret generic kurator-member1 --from-file=kurator-member1.config=/root/.kube/kurator-member1.config
+kubectl create secret generic kurator-member2 --from-file=kurator-member2.config=/root/.kube/kurator-member2.config
```

Comment:
It is unclear what this is operating to others.

AFAIK, there is a prerequisite, a host cluster is already installed. SO you should add the clusters setup as a  prerequisite